server/server.go

```diff
@@ -281,6 +288,15 @@ func (s *server) Boot() {
 			s.router.Path("/metrics").Handler(promhttp.Handler())
 		}
 
+		if s.enableDebugServer {
+			go func() {
+				s.logger.Log("level", "debug", "message", "running debug server at http://127.0.0.1:6060/debug")
+				// When net/http/pprof is imported, its init() registers /debug
+				// handles to DefaultServeMux automatically.
+				s.logger.Log("level", "debug", "message", "%#v", http.ListenAndServe("127.0.0.1:6060", nil))
```

Comment:
The message does not look right. You probably want to use the `fmt` package.

Reply:
Oh, good catch! Having written lately more `microerror.Maskf(....)` than logs, I got tripped. Fixing.